guidelines/round-table.md

```diff
@@ -0,0 +1,49 @@
+## Data Together Weekly Roundtable Call
+
+Once a week on **Mondays 5-5:30pm**, a rotating facilitator will lead a round table discussion of a Data Together-related topic, challenge, or question. We use a standing [Zoom call link](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201): [zoom.us/j/455833201](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201) and [GitHub `roundtable` issues](https://github.com/datatogether/datatogether/labels/roundtable) to run each call. **Check out [how to host](#how-to-host)** before running your first call.
+
+### Objectives
+
+- Check-in on ongoing work and potentially surface larger issues that need to be discussed in-depth
+- Allow initial feedback on developing ideas prior to opening a decision
+- Maintain momentum going between larger coordinating calls
+
+### Format
+
+Meetings will rotate individuals leading a round table around a specific topic, problem, or task. Scheduling topics will interleave implementation, governance, and topic area questions. Round table topics will be picked at least a week in advance.
+
+Example Round Table Topics:
+- checking in on decision-making process
+- reviewing website copy
+- asking about and open PR or design detail related to a reference implementation
+- identifying partnership & outreach opportunities
+
+#### Timing
+
+- **Mondays 5-5:30pm**
+- Secondary slot: Thursdays 5-5:30pm
```

Comment:
When do we use the secondary slot? Or maybe that doesn't matter! (I'm thinking it's data from the when2meet that you want to capture somewhere more permanent :slightly_smiling_face: )

Reply:
This was just data from when2meet. If Mondays ever become an issue :)